docker-compose.yml

```diff
@@ -14,11 +19,7 @@ services:
     volumes:
       - ${PWD}/data/mnesia:/home/aeternity/node/local/rel/aeternity/data/mnesia
       - ${PWD}/data/mdw.db:/home/aeternity/node/local/rel/aeternity/data/mdw.db
-      - ${PWD}/docker/aeternity.yaml:/home/aeternity/aeternity.yaml
+      - ${HOME}/aeternity.yaml:/home/aeternity/aeternity.yaml
       - ${PWD}/priv:/home/aeternity/node/ae_mdw/priv
-      - ${PWD}/test:/home/aeternity/node/ae_mdw/test
-      - ${PWD}/scripts/test.sh:/home/aeternity/node/ae_mdw/test.sh
-      - ${PWD}:/app
+      - ${PWD}/log:/home/aeternity/node/ae_mdw/log
     expose: [ 3013, 3014, 3113, 4001, 4000 ]
-    environment:
```

Comment:
Would it make sense to add `- MIX_ENV=prod` here?

Reply:
i think defaults are not needed (unless there is a possibility that someone change them)

Reply:
To be done when we move to releases in order to allow adding dynamic prod configs.

Reply:
> i think defaults are not needed (unless there is a possibility that someone change them), or if it is not the default of course

The default right now is `dev` and that's what's being run right now in prod

Reply:
then it makes sense to add it

Reply:
I guess it also makes sense to add it to the builder https://github.com/aeternity/ae_mdw/blob/master/.github/workflows/dockerhub.yml#L38

Reply:
The point is that `dev.exs` has always being used and I will start working on the Elixir releases that will change the way config works. I will provide an initial prod config though.